DOL.WHD.Section14c.Web/src/modules/components/attachmentField/attachmentFieldTemplate.html

```diff
@@ -1,14 +1,51 @@
-<div ng-show="attachmentId == undefined">
+<div ng-show="!restrictUpload">
   <div class="dol-file-upload-ui">
     <div aria-hidden="true" class="usa-button-primary">Browse</div>
-    <input class="dol-file-upload-input" id="{{inputId}}" class="upload-file-input" type="file"  aria-describedby="fileUploadDesc_{{::$id}}" onchange="angular.element(this).scope().vm.onAttachmentSelected(this)" accept=".doc,.docx,.xls,.xlsx,.pdf,.jpg,.jpeg,.png">
+    <input class="dol-file-upload-input" id="{{inputId}}" type="file" aria-describedby="fileUploadDesc_{{::$id}}" onchange="angular.element(this).scope().vm.onAttachmentSelected(this)" accept=".pdf,.jpg,.jpeg,.png,.csv">
     <div class="dol-file-upload-focus"></div>
-  </div>  
-  <p class="underlabel" id="fileUploadDesc_{{::$id}}">File types accepted: PDF, Word, Excel, JPG, PNG</p>
+  </div>
+  <p class="underlabel" id="fileUploadDesc_{{::$id}}">File types accepted: .pdf, .jpg, .jpeg, .png, .csv</p>
+  <p>
+    Need to convert a Word or Excel file to PDF? <a href="https://support.office.com/en-us/article/Save-or-convert-to-PDF-d85416c5-7d77-4fd6-a216-6f4bf7c7c110" target="blank">Follow these instructions.</a>
+  </p>
 </div>
-<div ng-if="attachmentId != undefined">
-  <a ng-href="{{vm.apiService.attachmentApiURL}}{{vm.stateService.ein}}/{{attachmentId}}?access_token={{vm.stateService.access_token}}" id="{{attachmentId}}">
-    <span>{{attachmentName}}</span>
-  </a>
-  <button class="usa-button-secondary" ng-click="vm.deleteAttachment(attachmentId)">Delete</button>
+<div >
+  <ul class="dol-uploaded-files">
+    <li ng-repeat="attachment in formData[modelPrefix][inputId]">
+      <strong>Uploaded:</strong><a ng-href="{{vm.apiService.attachmentApiURL}}{{vm.stateService.ein}}/{{attachment.attachmentId}}?access_token={{vm.stateService.access_token}}" id="{{attatchment.attachmentId}}"><span>{{attachment.attachmentName}}</span></a>
+      <button aria-controls="modal" class="red-button modal-trigger" ng-click="vm.showModal(attachment.attachmentId, attachment.attachmentName)">Delete<span class="hide"> {{attachment.attachmentName}}</span></button>
+    </li>    
+  </ul>
+  <div>
+    <div ng-show="upload.status == 'Uploading'" class="dol-feedback-progress">
+        <p class="dol-feedback-message">
+          <strong>Uploading:</strong> {{attachmentName}}
+        </p>
+    </div>
+    <div ng-show="upload.status == 'Invalid'" class="dol-feedback-error">
+        <p class="dol-feedback-message">
+          <strong>{{upload.message}}</strong> <span>{{attachmentName}} was not updated.</span><span ng-show="upload.message === 'File Size too large.'"> Files cannot be larger than 5MB.</span><span ng-show="upload.message === 'Invalid File Type.'"> Accepted file types are: PDF, Word, Excel, JPG, PNG</span>
```

Comment:
Should this message be `... was not uploaded`?  It says "updated" now, which might be correct depending on context.  Just asking to be sure!